scripts/funcotator/data_sources/getGencode.sh

```diff
@@ -89,9 +88,10 @@ function createConfigFile() {
 
     local dataSourceName=$1
     local version=$2
-    local srcFile=$3
-    local originLocation=$4
-    local fastaPath=$5
+    local refVersion=$3
+    local srcFile=$4
+    local originLocation=$5
+    local fastaPath=$6
```

Comment:
This script has a lot of arguments -- should ideally document them in a comment at the top.

Reply:
No problem.  I'll add in comments.